tests/test_env.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+"""Tests for context managers related to environment variables."""
```

Comment:
Need to update comment (it's not a context manager any more).

Reply:
Fixed in https://github.com/oracle/macaron/pull/389/commits/e5d50e65d823731d54b72edc5787b3171ddb6663